packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -0,0 +1,224 @@
+---
+title: OIN submission requirements
+meta:
+  - name: description
+    content: Use this guide to prepare all the requirements prior to submitting your integration to Okta for publication in the Okta Integration Network.
+layout: Guides
+---
+
+This guide provides you with a list of requirements you need prepare prior to submitting your integration for publication in the [Okta Integration Network (OIN)](https://www.okta.com/integrations/).
+
+---
+
+**Learning outcome**
+
+* Understand the requirements necessary for the submission process using the OIN Manager.
+
+**What you need**
+
+* a built and tested OIN integration that is ready for Okta verification
+
+---
+
+## Overview
+
+Prior to using the [OIN Manager](https://oinmanager.okta.com) to submit your OIN integration, you need to review the following guidelines to prepare for the artifacts requested during the submission process:
+
+* [Logo guidelines](#logo-guidelines)
+* [App description guidelines](#app-description-guidelines)
+* [Use case guidelines](#use-case-guidelines)
+* [Test account guidelines](#test-account-guidelines)
+* [Customer configuration document guidelines](#customer-configuration-document-guidelines)
+
+Before you follow the submission guidelines, ensure that your integration does not contain [unsupported OIN features](#unsupported-oin-features).
```

Comment:
what does this line mean?

Reply:
oh sorry, it's right below 😆 

I personally don't think this belongs in this document. Glancing at it, all of them are SSO. All except SWA are OIDC related. I think we could put this in SSO/OIDC build documentation and more as a FAQ / disclaimers in context or as a footnote.

Reply:
Yes, we should also add it in the OIN SSO documents. One of the complaints from dev-support is that this information is hidden in the various OIN documents and we need to inform potential builders of these limitations early on in the process. I'll add it to the bottom of this page for now and make it a bulleted list (instead of box notes). We can move this content to other pages as we work on them.